source/_integrations/sonarr.markdown

```diff
@@ -16,4 +16,21 @@ The `Sonarr` integration pulls data from a given [Sonarr](https://sonarr.tv/) in
 
 ## Configuration
 
-Go to the integrations page in your configuration and click on new integration -> Sonarr.
+Go to the Configuration > Integrations page and click on the plus '+' to add a new integration. Select Sonarr. Enter your Sonarr connection information and click Submit. 
```

Comment:
We can now use a new plugin called `configuration_basic` 

Checkout the documentation for the Lovelace Weather card on how to use this plugin

Something along the lines of this
```markdown
{% configuration_basic %}

(Here you will place your variables

{% endconfiguration_basic %}
```